railties/test/generators/scaffold_generator_test.rb

```diff
@@ -425,6 +425,18 @@ def test_scaffold_generator_no_stylesheets
     assert_no_file "app/assets/stylesheets/posts.css"
   end
 
+  def test_scaffold_generator_alias_css_stylesheets_engine
+    run_generator [ "posts", "--se=css" ]
+    assert_file "app/assets/stylesheets/posts.css"
+    assert_no_file "app/assets/stylesheets/posts.scss"
+  end
+
+  def test_scaffold_generator_alias_nocss_stylesheets_engine
+    output = run_generator [ "posts", "--se=NOCSS" ]
+    assert_no_file "app/assets/stylesheets/posts.css"
+    assert_match(/NOCSS \[not found\]/, output)
+  end
```

Comment:
I'm not sure this test adds anything to this PR. It's also the only one in ScaffoldGeneratorTest that test for not found.

Reply:
Is a reverse way for `test_scaffold_generator_alias_css_stylesheets_engine` I'll try to use a default scss but for this test don't have sense.